middleware/cors.js

```diff
@@ -13,6 +13,17 @@ var cors = require('cors');
 
 exports.init = function(app, config, logger, callback) {
     var cfg = config.get('cors');
+
+    if (cfg.origin instanceof Array) {
+        // Convert any CORS origin defined with regex (i.e. starts with '^')
+        // from JSON string into an actual RegExp object.
+        for (var i = 0; i < cfg.origin.length; i++) {
+            if (cfg.origin[i].startsWith('^')) {
```

Comment:
`String.prototype.startsWith` is not supported by older versions of node.js which BOS supports